docs/src/index.md

```diff
@@ -1,160 +1,263 @@
-# Catalyst.jl for Reaction Network Modeling
+# [Catalyst.jl for Reaction Network Modeling](@id doc_index)
 
-Catalyst.jl is a symbolic modeling package for analysis and high performance
+Catalyst.jl is a symbolic modeling package for analysis and high-performance
 simulation of chemical reaction networks. Catalyst defines symbolic
-[`ReactionSystem`](@ref)s, which can be created programmatically or easily
-specified using Catalyst's domain specific language (DSL). Leveraging
-[ModelingToolkit.jl](https://docs.sciml.ai/ModelingToolkit/stable/) and
-[Symbolics.jl](https://docs.sciml.ai/Symbolics/stable/), Catalyst enables
+[`ReactionSystem`](@ref)s,
+which can be created programmatically or easily
+specified using Catalyst's domain-specific language (DSL). Leveraging
+[ModelingToolkit.jl](https://github.com/SciML/ModelingToolkit.jl) and
+[Symbolics.jl](https://github.com/JuliaSymbolics/Symbolics.jl), Catalyst enables
 large-scale simulations through auto-vectorization and parallelism. Symbolic
```

Comment:
I've always thought this comment about "auto-vectorization and parallelism" was a bit disingenuous. We don't really do/expose either as far as I am aware... Not sure if we should change it somehow. @ChrisRackauckas any thoughts?